<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtMaziakMenuDialog2</class>
 <widget class="QWidget" name="QtMaziakMenuDialog2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Maziak</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/R.png</normaloff>:/images/R.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_top" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget_top_left" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_top_middle" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_top_right" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_easy" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_medium" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_hard" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_spacer_bottom" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_start" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_start_retro" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_instructions" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_about" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_quit" native="true"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
